src/views/CardStack.js

```diff
@@ -65,7 +69,7 @@ type Props = {
 /**
  * The duration of the card animation in milliseconds.
  */
-const ANIMATION_DURATION = 200;
+const ANIMATION_DURATION = 250;
```

Comment:
Sure, feel free to just update the comment and mention the conflict with TransitionConfigs